README.md

```diff
@@ -1,40 +1,81 @@
+	text2 = "Lorem dolor sit amet."
+)
+
+func main() {
+	dmp := diffmatchpatch.New()
+
+	diffs := dmp.DiffMain(text1, text2, false)
+
+	fmt.Println(dmp.DiffPrettyText(diffs))
+}
+```
+
+## I found a bug or missing a feature in go-diff
+
+Please make sure to have the latest version of go-diff. If the problem still persists go through the [open issues](https://github.com/sergi/go-diff/issues) in the tracker first. If you cannot find your request just open up a [new issue](https://github.com/sergi/go-diff/issues/new).
+
+## How to contribute?
+
+You want to contribute to go-diff? GREAT! If you are here because of a bug you want to fix or a feature you want to add, you can just read on. Otherwise we have a list of [open issues in the tracker](https://github.com/sergi/go-diff/issues). Just choose something you think you can work on and discuss your plans in the issue by commenting on it.
+
+Please make sure that every behavioral change is accompanied by test cases. Additionally, every contribution must pass the `lint` and `test` Makefile targets which can be run using the following commands in the repository root directory.
+
+```bash
+make lint
+make test
+```
+
+After your contribution passes these commands, [create a PR](https://help.github.com/articles/creating-a-pull-request/) and we will review your contribution.
+
+## Origins
 
-    go get github.com/sergi/go-diff/diffmatchpatch
+go-diff is a Go language port of Neil Fraser's google-diff-match-patch code. His original code is available at [http://code.google.com/p/google-diff-match-patch/](http://code.google.com/p/google-diff-match-patch/).
 
 ## Copyright and License
 
-The original Google Diff, Match and Patch Library is licensed under
-the [Apache License 2.0](http://www.apache.org/licenses/LICENSE-2.0).
-The full terms of that license are included here in the
-`APACHE-LICENSE-2.0` file.
+The original Google Diff, Match and Patch Library is licensed under the [Apache License 2.0](http://www.apache.org/licenses/LICENSE-2.0). The full terms of that license are included here in the `APACHE-LICENSE-2.0` file.
 
 Diff, Match and Patch Library
 
-  Written by Neil Fraser
-  Copyright (c) 2006 Google Inc.
-  <http://code.google.com/p/google-diff-match-patch/>
+> Written by Neil Fraser
+> Copyright (c) 2006 Google Inc.
+> <http://code.google.com/p/google-diff-match-patch/>
 
-This Go version of Diff, Match and Patch Library is licensed under
-the [MIT License](http://www.opensource.org/licenses/MIT) (a.k.a.
-the Expat License) which is included here in the `LICENSE` file.
+This Go version of Diff, Match and Patch Library is licensed under the [MIT License](http://www.opensource.org/licenses/MIT) (a.k.a. the Expat License) which is included here in the `LICENSE` file.
 
 Go version of Diff, Match and Patch Library
 
-  Copyright (c) 2012 Sergi Mansilla <sergi.mansilla@gmail.com>
-  <https://github.com/sergi/go-diff>
+> Copyright (c) 2012 Sergi Mansilla <sergi.mansilla@gmail.com>
```

Comment:
Hey! Feel free to add yourself as a contributor. You've done a lot of work on this library.